os/arch/arm/src/imxrt/imxrt_start.c

```diff
@@ -316,8 +316,8 @@ static void go_os_start(void *pv, unsigned int nbytes)
  *   Make configuration to use the entire 512KB FlexRAM as OCRAM
  *
  ****************************************************************************/
-
-void imxrt_configure_ocram()
+#ifndef CONFIG_ARMV7M_DTCM
```

Comment:
Is it right? not CONFIG_IMXRT_DTCM?

Reply:
Yes.. Either dtcm or ocram is configured. 
However, I modified the code slightly as below:

`#ifdef DTCM'
'void imxrt_configure_dtcm()'
'#else'
'void imxrt_configure_ocram()'